go/libkb/client.go

```diff
@@ -350,17 +350,18 @@ func NewInstrumentedTransport(g *GlobalContext, tagger func(*http.Request) strin
 	}
 }
 
-func (i *InstrumentedTransport) getGzipWriter(writer io.Writer) (*gzip.Writer, func()) {
+func (i *InstrumentedRoundTripper) getGzipWriter(writer io.Writer) (*gzip.Writer, func()) {
 	gzipWriter := i.gzipPool.Get().(*gzip.Writer)
 	gzipWriter.Reset(writer)
 	return gzipWriter, func() {
 		i.gzipPool.Put(gzipWriter)
 	}
 }
 
-func (i *InstrumentedTransport) RoundTrip(req *http.Request) (resp *http.Response, err error) {
+func (i *InstrumentedRoundTripper) RoundTrip(req *http.Request) (resp *http.Response, err error) {
 	record := rpc.NewNetworkInstrumenter(i.G().RemoteNetworkInstrumenterStorage, i.tagger(req))
-	resp, err = i.Transport.RoundTrip(req)
+	req.Close = true
```

Comment:
Why do we need a `ClosingRoundTripper` if this closes?

Reply:
I thought I got rid of this...